src/main/java/com/example/TournamentProject/TournamentManagement.java

```diff
@@ -0,0 +1,77 @@
+package com.example.TournamentProject;
+
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class TournamentManagement {
+
+    public void getTournamentName() {
```

Comment:
`getTournamentName` - means that as the result of this method I will GET tournament name
If you want just print something into console - print it, but do not call method, which printing with GET prefix
and one more - why you are printing something into console from `TournamentManagement` service? For doing actions with console - you have `Console` service